src/main/java/com/durov/maks/cinema/util/HashUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.durov.maks.cinema.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.apache.log4j.Logger;
+
+public class HashUtil {
+    private static final Logger LOGGER =
```

Comment:
```suggestion
    private static final Logger logger =
```
see logger naming here) http://www.quizful.net/test

Reply:
got it)